feeds/wlan-ap/opensync/patches/19-events-protobuf-definitions.patch

```diff
@@ -1,7 +1,5 @@
-Index: opensync-2.0.5.0/interfaces/opensync_stats.proto
-===================================================================
---- opensync-2.0.5.0.orig/interfaces/opensync_stats.proto
-+++ opensync-2.0.5.0/interfaces/opensync_stats.proto
+--- a/interfaces/opensync_stats.proto
```

Comment:
Is the change to the path here significant?  Applies to both patches.

Reply:
I think this is because I have changed the existing patch. The regular quilt process. Doesn't seem to be an issue.